webapp/pages/terms-and-conditions.vue

```diff
@@ -1,14 +1,72 @@
 <template>
   <div>
     <ds-space>
-      <ds-heading tag="h2">{{ $t('site.termsAc') }}</ds-heading>
+      <ds-heading tag="h2">{{ $t('site.termsAc') }} ( vom: {{ latestDate_TaC }})</ds-heading>
     </ds-space>
+    <div v-if="confirmed === true">
+      <ds-section class="hasConfirm" primary>
```

Comment:
```suggestion
      <ds-section primary>
```
Unnecessary class

Reply:
I think there is a redundancy here. What about using class bindings?

```suggestion
      <ds-section :primary="confirmed" :secondary="!confirmed">
```
This should make 90% of the template code obsolete.